source/sundials/arkode.cc

```diff
@@ -612,13 +631,15 @@ namespace SUNDIALS
 #  else
             sun_mass_linear_solver =
               SUNLinSol_SPGMR(y_template,
-                              PREC_NONE,
+                              SUN_PREC_NONE,
```

Comment:
I suppose that `SUN_PREC_NONE` has also been introduced in 6.0 already?

Reply:
Indeed.